plugins/trezor/trezor.py

```diff
@@ -17,23 +17,20 @@ def __init__(self, *args):
             import trezorlib
             import trezorlib.ckd_public
             import trezorlib.transport_hid
+            import trezorlib.messages
             self.client_class = client.TrezorClient
             self.ckd_public = trezorlib.ckd_public
-            self.types = trezorlib.client.types
-            self.DEVICE_IDS = trezorlib.transport_hid.DEVICE_IDS
+            self.types = trezorlib.messages
+            self.DEVICE_IDS = (trezorlib.transport_hid.DEV_TREZOR1, trezorlib.transport_hid.DEV_TREZOR2)
             self.libraries_available = True
         except ImportError:
             self.libraries_available = False
         TrezorCompatiblePlugin.__init__(self, *args)
 
-    def hid_transport(self, pair):
+    def hid_transport(self, device):
         from trezorlib.transport_hid import HidTransport
-        return HidTransport(pair)
+        return HidTransport.find_by_path(device.path)
 
     def bridge_transport(self, d):
         from trezorlib.transport_bridge import BridgeTransport
         return BridgeTransport(d)
-
-    def get_coin_name(self):
-        # No testnet support yet
-        return "Bcash"
```

Comment:
Uhh where did get_coin_name go?

Reply:
It moved to the base class in plugin.py.  I moved it to make it more similar to the electrum code base.

Reply:
Ah sorry, missed that.